docs/python/debugging.md

```diff
@@ -95,7 +95,7 @@ Specifies how program output is displayed.
 
 ### `cwd`
 
-Specifies the current working directory for the program being debugged, defaulting to the current folder. The standard configuration uses `${workspaceFolder}` to use the project folder.
+Specifies the current working directory for the program being debugged. The standard configuration uses `${workspaceFolder}` to use the project folder. If is not specified, `cwd` defaults to the current folder open in VS Code.
```

Comment:
Would "If not specified, .." read better? Or "If it is not specified, ..."

Reply:
Doh. Hate it when that happens.

Reply:
Does "_current folder_" mean the folder that is opened via **File -> Open Folder...** ? Or this could be the folder-child of the project, where the py file is?

Thank you, if you can clarify this question as well.

Reply:
It's the folder containing the Python file that you started in the debugger. I'll clarify that in the next set of changes, along with an example. Thanks for bringing that up, and I appreciate the push for clarity. :)

Reply:
Correction, the default for cwd if omitted is always ${workspaceFolder}, which is what we had there originally. The next set of updates clarifies this and adds examples (see PR #1532).